proposals/lip-0051.md

```diff
@@ -511,6 +512,9 @@ def verify(trs: Transaction) -> None:
     if receivingChainID == OWN_CHAIN_ID:
         raise Exception("Receiving chain cannot be the sending chain.")
 
+    if not Interoperability.isChannelActive(receivingChainID):
```

Comment:
Just a note, since in mainchain we need `timestamp` for `isLive` function being called from `isChannelActive` function, we pass it as an argument to it in SDK, on mainchain we call `isChannelActive(receivingChainID, timestamp)` and on sidechain `isChannelActive(receivingChainID)` so in token module we make `timestamp` arg optional similar to `interop.send()` where also we need timestamp

Reply:
Thanks for the clarification of the SDK implementation.